lxd/instance/drivers/driver_common.go

```diff
@@ -1006,3 +1007,13 @@ func (d *common) getRootDiskDevice() (string, map[string]string, error) {
 
 	return name, configuration, nil
 }
+
+// resetInstanceID generates a new UUID and puts it in volatile.
+func (d *common) resetInstanceID() error {
+	err := d.VolatileSet(map[string]string{"volatile.cloud-init.instance-id": uuid.New()})
+	if err != nil {
+		return fmt.Errorf("Failed to set volatile.cloud-init.instance-id volatile key: %w", err)
```

Comment:
@stgraber Isn't it obvious that `volatile.cloud-init.instance-id` is volatile? ;) Perhaps we can drop the word `volatile`.